man/man1/valid_plugins.1

```diff
@@ -0,0 +1,14 @@
+./" Man page for valid_plugins
+.TH man 1 "26 November 2020" "1.0" "valid_plugins man page"
+.SH NAME
+valid_plugins -/ Returns valid plugins
```

Comment:
"Returns valid plugins from a configuration file. A valid plugin matches this regex: `^Plugin{1} [^ ]* inputfile{1} [^ ]* outputfile{1} [^ ]*`"